core/vm/eof.go

```diff
@@ -105,6 +108,46 @@ func (meta *functionMetadata) checkStackMax(stackMax int) error {
 	return nil
 }
 
+func (c *Container) codeSectionSize(s int) int {
```

Comment:
I don't think it makes sense to use `codeSectionSize` over just `len(codeSectionBytes)` since they are almost equally fast in go:

```go
func BenchmarkBytes(b *testing.B) {
	slice := make([]byte, 100)
	getBytes := func() []byte {
		return slice[0:10]
	}
	for i := 0; i < b.N; i++ {
		_ = len(getBytes())
	}
}

func BenchmarkLength(b *testing.B) {
	getBytes := func() int {
		return 10 - 0
	}
	for i := 0; i < b.N; i++ {
		_ = getBytes()
	}
}
```

BenchmarkBytes-24    	1000000000	         0.4572 ns/op	       0 B/op	       0 allocs/op
BenchmarkLength-24    	1000000000	         0.2184 ns/op	       0 B/op	       0 allocs/op